### ISSUE TITLE
Feature: Split the two-page picture from the middle

### ISSUE THREAD

**Issue by jim-jay** [KotatsuApp/Kotatsu#352]
### Describe your suggested feature

There are some images that contain two pages, horizontal long and vertical short, and the current app does not seem to be able to show this situation well.

Now it looks like this
![94eabc2a5060515dc29e91fba14b702](https://user-images.githubusercontent.com/30513257/234148528-8651bc25-25b8-4189-8f4c-aaf5f0e70573.jpg)

And we just need to read the right part first
![1e25bfee3043d0cba0d0deda3389bcc](https://user-images.githubusercontent.com/30513257/234148643-0c7411dd-8d66-487e-9226-0e427b6d853a.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Koitharu** [comment]
I guess this is one image, right?

**Comment by jim-jay** [comment]
> I guess this is one image, right?

That's right. 👌

**Comment by Koitharu** [comment]
I suppose it is not possible to split existing images into multiple parts.

**Comment by glasszay** [comment]
> I suppose it is not possible to split existing images into multiple parts.

It has been done before by the tachiyomi community, called **split pages**. Not sure if it's possible to implement though.

**Comment by jim-jay** [comment]
> > I suppose it is not possible to split existing images into multiple parts.
> 
> It has been done before by the tachiyomi community, called **split pages**. Not sure if it's possible to implement though.

Yes, but I'm not good at Android development, so I could not help more.
